Objects/dictobject.c

```diff
@@ -3697,6 +3924,32 @@ dictiter_reduce(dictiterobject *di, PyObject *Py_UNUSED(ignored))
     return Py_BuildValue("N(N)", _PyObject_GetBuiltin("iter"), list);
 }
 
+PyTypeObject PyDictRevIterItem_Type = {
+    PyVarObject_HEAD_INIT(&PyType_Type, 0)
+    "dict_reverseitemiterator",
+    sizeof(dictiterobject),
+    .tp_dealloc = (destructor)dictiter_dealloc,
+    .tp_getattro = PyObject_GenericGetAttr,
```

Comment:
Can't we skip defining `.tp_getattro`?

Reply:
Hi @methane, thanks for the comments. I'm fixing the other issues. For this attribute, I'm not sure to understand precisely how `tp_getattro` is used and set it here because it was set on `PyDictKeys_Type`. As I understand it, it's the implementation of `getattr`. Is that right?

Reply:
When NULL is set to members, PyType_Ready fills them to default values.
And `PyObject_GenericGetAttr` is the default value.

When we skip specifying all defaults explicitly, programmers can focus on
meaningful part of the code.

Reply:
Thank you so much for the clarification, I read `PyType_Ready` definition and it all makes sense now.

The documentation at https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_getattro for this field says 

>An optional pointer to the get-attribute function.
> 
> The signature is the same as for PyObject_GetAttr(). It is usually convenient to set this field to PyObject_GenericGetAttr(), which implements the normal way of looking for object attributes.

Would it makes sense to propose a change from *"It is usually convenient to set this field to PyObject_GenericGetAttr(), which implements the normal way of looking for object attributes."* to something like *"If not set, `PyType_Ready` will set this field to PyObject_GenericGetAttr(), which implements the normal way of looking for object attributes."* for this field and the ones alike?